2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/tesis/cloud-classification/code/wandb/settings
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_225410-2sq34yaq/logs/debug.log
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_225410-2sq34yaq/logs/debug-internal.log
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f158461eee0>
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:init():439] calling init triggers
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'GATConvGNN', 'loss': 'LogeLoss', 'optim': 'SGD', 'dataset': 'GCD', 'epochs': 30}
2022-03-31 22:54:10,916 INFO    MainThread:18344 [wandb_init.py:init():492] starting backend
2022-03-31 22:54:10,917 INFO    MainThread:18344 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-31 22:54:10,923 INFO    MainThread:18344 [backend.py:ensure_launched():219] starting backend process...
2022-03-31 22:54:10,935 INFO    MainThread:18344 [backend.py:ensure_launched():224] started backend process with pid: 18371
2022-03-31 22:54:10,938 INFO    MainThread:18344 [wandb_init.py:init():501] backend started and connected
2022-03-31 22:54:10,959 INFO    MainThread:18344 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-31 22:54:10,959 INFO    MainThread:18344 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-31 22:54:10,959 INFO    MainThread:18344 [wandb_init.py:init():565] updated telemetry
2022-03-31 22:54:10,982 INFO    MainThread:18344 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-31 22:54:11,762 INFO    MainThread:18344 [wandb_run.py:_on_init():1759] communicating current version
2022-03-31 22:54:11,847 INFO    MainThread:18344 [wandb_run.py:_on_init():1763] got version response 
2022-03-31 22:54:11,848 INFO    MainThread:18344 [wandb_init.py:init():625] starting run threads in backend
2022-03-31 22:54:13,433 INFO    MainThread:18344 [wandb_run.py:_console_start():1733] atexit reg
2022-03-31 22:54:13,434 INFO    MainThread:18344 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-31 22:54:13,435 INFO    MainThread:18344 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-31 22:54:13,437 INFO    MainThread:18344 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-31 22:54:13,437 INFO    MainThread:18344 [wandb_init.py:init():664] run started, returning control to user process
2022-03-31 22:54:13,446 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:13,446 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:13,478 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:13,479 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:13,479 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:13,500 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:13,501 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:13,501 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:13,522 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,652 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,653 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,674 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,675 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,675 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,697 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,697 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,698 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,739 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,741 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,741 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,772 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,772 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,773 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,810 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 22:54:15,813 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 22:54:15,813 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 22:54:15,843 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:17:41,494 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:17:41,494 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:19:18,800 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:19:18,809 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:19:18,814 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:19:26,097 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:19:26,106 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:19:26,107 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:19:34,346 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:19:34,356 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:19:34,356 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:19:38,631 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:19:38,641 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:19:38,641 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:19:54,048 INFO    MainThread:18344 [wandb_init.py:_resume_backend():327] resuming backend
2022-03-31 23:19:54,051 INFO    MainThread:18344 [wandb_init.py:_pause_backend():318] pausing backend
2022-03-31 23:19:54,051 INFO    MainThread:18344 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 23:20:58,408 INFO    MainThread:18344 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-31 23:20:58,410 INFO    MainThread:18344 [wandb_run.py:_restore():1674] restore
